lib/public-stats.js

```diff
@@ -282,15 +284,84 @@ const updateRetrievalTimings = async (pgClient, committees) => {
  * @param {(minerId: string, cid: string) => (string[] | undefined)} findDealClients
  */
 export const updateDailyClientRetrievalStats = async (pgClient, committees, findDealClients) => {
-  const retrievalStatsPerClient = new Map()
+  const flatStats = buildPerPartyStats(committees, findDealClients, 'client')
```

Comment:
```suggestion
  const stats = buildPerPartyStats(committees, findDealClients, 'client')
```

Flattening the stats is an implementation detail of `buildPerPartyStats` and doesn't need to be communicated here (it confused me reviewing the code)